### PR TITLE
View participant list of active quest

### LINKED ISSUE #10508: can't view participant list of active quest
As reported by Ireanna (267135db-e7f8-42d9-bed2-f84b4261b64e):

**"once a quest is launched, the participant list can't be viewed anymore. this has been an issue since the whole big redesign of the site I think, just never got around to reporting it. there is the word "Participants" but clicking it does nothing."**

They're correct that it's been happening since the redesign and I recall that we were aware of it at the time but it looks like we never created an issue for it.

Clicking on "Participants" (shown below) should open a modal that shows the names of all the people who are participating in the quest. Ideally, clicking on each name should open that player's profile, but that's not essential.

image

### PR DESCRIPTION
**Pull request by thefifthisa**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #10508 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

Added a modal that displays a list of quest participants. (I adapted the design from the party members modal.)

questparticipants

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID: 9ca2bb65-ce6c-4e4c-a367-5ef9b5c7421e

### CODE REVIEW

File: website/client/components/groups/participantListModal.vue [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  components: {
+    MemberDetails,
+  },
+  computed: {
+    ...mapGetters({
+      partyMembers: 'party:members',
+    }),
+    participants () {
+      let partyMembers = this.partyMembers || [];
+      return partyMembers.map(member => {
```
Comment (negue) [inline website/client/components/groups/participantListModal.vue]: you could use `filter` instead of `.map`ping first

**Comment by negue** [comment]
Tested it, works good :) 👍 

@Tressley what do you think about the button / participant list

File: website/client/components/groups/participantListModal.vue (hunk 1 above)
Comment (thefifthisa) [reply website/client/components/groups/participantListModal.vue]: Got it! Realized I don't need to map at all 😅

**Comment by TheHollidayInn** [comment]
Thanks for the PR! Can we change the button to a link for now? We can come back to the design

**Comment by thefifthisa** [comment]
Sure thing! Is this okay?

link

**Comment by TheHollidayInn** [comment]
Awesome! Sorry, one more thing. Can you make the link color the same as the text was? Or the same as the Feral Dust Bunnies in the image you linked?

**Comment by thefifthisa** [comment]
Which one do you prefer?

link2
link

**Comment by TheHollidayInn** [comment]
The bottom, please

**Comment by thefifthisa** [comment]
Got it! Although, is there a better way to style a link as another element aside from explicitly changing the inline style like I did here?

**Comment by TheHollidayInn** [comment]
Sweet :D And yes, you can add a class/id to the item then add your css styles inside this tag: https://github.com/HabitRPG/habitica/pull/10531/files#diff-1aa6495987661eb4bb432587a6dbafa1L65

**Comment by thefifthisa** [comment]
Done!

**Comment by Alys** [comment]
@thefifthisa I'm really sorry about this but in the latest Bailey message, I've been given credit for making this change instead of you. :(  I think the confusion would have crept in because I did a small addition later to apply your same change to collection quests (

**Comment by Alys** [comment]
@thefifthisa I'm really sorry about this but in the latest Bailey message, I've been given credit for making this change instead of you. :( I think the confusion would have crept in because I did a small addition later to apply your same change to collection quests (https://github.com/HabitRPG/habitica/pull/10568).  I've already fixed the attribution in the wiki's What's New page (https://habitica.wikia.com/wiki/Whats_New_2018#New.21_Quest_Partipant_List ), and I'm about to post to the Tavern and the Aspiring Helpers guilds too. I'm also asking the staff to edit the Bailey message, if they can get to it before a new one replaces it.

Thank you for your work to fix this issue. We really do appreciate it!

**Comment by thefifthisa** [comment]
@Alys Aw no worries at all! Thanks for letting me know and I really appreciate the correction 😊

**Pull request merged**